src/bp-search/classes/class-bp-search.php

```diff
@@ -886,7 +886,7 @@ public function print_tabs() {
 				$class = $item == $this->search_args['search_subset'] ? 'active current' : '';
 				// this filter can be used to change display of 'posts' to 'Blog Posts' etc..
 
-				$label = isset( $search_items[ $item ] ) ? $search_items[ $item ] : $item;
+				$label = isset( $search_items[ $item ] ) ? $search_items[ $item ] : ucwords( $item );
```

Comment:
revert the changes and add the filter on the video component just we did on the discussions.
`add_filter( 'bp_search_label_search_type', 'bp_document_search_label_search' );`